doc/api/tty.md

```diff
@@ -82,12 +82,15 @@ or `writeStream.rows` properties have changed. No arguments are passed to the
 listener callback when called.
 
 ```js
+process.stdin.setRawMode(true);
```

Comment:
I think what @bzoz was saying in https://github.com/nodejs/node/issues/13197#issuecomment-307073097 is that this line is needed on Windows, which implies it isn't needed on other platforms. 

The code snippet without this sample works reliably in Unix, and even with this sample it's not reliable on Windows (https://github.com/nodejs/node/issues/13197#issuecomment-307075204), so I think it's probably best to just not include this (especially if this leads to people not being able to Ctrl+C their program).